plugins/TagFix_IntervalConditional.py

```diff
@@ -0,0 +1,77 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyright Adrien Pavie 2019                                           ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+from plugins.Plugin import Plugin
+
+try:
+    from transporthours.main import Main
+    module_transporthours = True
+except ImportError as e:
+    print(e)
+    module_transporthours = False
+
+class TagFix_IntervalConditional(Plugin):
+
+    def init(self, logger):
+        if not module_transporthours:
+            return False
+        Plugin.init(self, logger)
+        self.errors[32502] = {"item": 3250, "level": 3, "tag": ["value", "fix:chair"], "desc": T_(u"Invalid Conditional Intervals")}
```

Comment:
maybe add `public_transport` to the tag list

Reply:
Done